boards/weact/bluepillplus_ch32v203/doc/index.rst

```diff
@@ -0,0 +1,85 @@
+.. zephyr:board:: bluepillplus_ch32v203
+
+Overview
+********
+
+The `WeActStudio`_ Blue Pill Plus CH32V203 hardware provides support for QingKe 32-bit RISC-V4B
+processor and the following devices:
+
+* CLOCK
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+
+The board is equipped with two LEDs and two Buttons.
```

Comment:
nit: it's two LEDs, but only one is controllable, and 3 buttons.

Perhaps say "with a user LED and user button in addition to a power LED, reset button, and boot select button."

also s/Button/button/

Reply:
I forgot to update that part, thats from linkw.

Reply:
Done.